_stbt/core.py

```diff
@@ -1775,31 +1816,64 @@ def _find_match(image, template, match_parameters, imglog):
             else:
                 roi_mask = cv2.pyrUp(roi_mask)
 
-        matched, best_match_position, certainty, roi_mask = _match_template(
-            image_pyramid[level], template_pyramid[level], match_parameters,
+        heatmap = _match_template(
+            image_pyramid[level], template_pyramid[level], method,
             roi_mask, level, imglog)
 
+        # Relax the threshold slightly for scaled-down pyramid levels to
+        # compensate for scaling artifacts.
+        threshold = max(
+            0,
+            match_parameters.match_threshold - (0.2 if level > 0 else 0))
```

Comment:
It isn't new -- the old one is [here](https://github.com/stb-tester/stb-tester/pull/212/files#diff-b8db1b2ea3c8e5386ca27ac571ea7aa6L1788).

The comment explaining it is new.